src/clr-angular/data/stack-view/stack-block.ts

```diff
@@ -131,4 +128,12 @@ export class ClrStackBlock implements OnInit {
   get onStackLabelFocus(): boolean {
     return this.expandable && !this.expanded && this.focused;
   }
+
+  get ariaExpanded(): string {
+    if (!this.expandable) {
+      return null;
+    } else {
+      return this.expanded ? 'true' : 'false';
```

Comment:
This doesn't need to be a string, I believe. So `return this.expanded` could work as well.